index.js

```diff
@@ -162,7 +162,10 @@ const parseLang = (str) => {
     .replace(/\}$/, '');
 
   const attrs = selectors.length ? selectorToAttrs(selectors) : {};
-  const className = classNames(lang ? `language-${lang}` : '', attrs.class);
+  const className = classNames(
+    lang ? `language-${lang}` : 'language-',
```

Comment:
any reason why not an empty string?

Reply:
I agree that it looks like a bad thing but sadly prism themes often use this kind of checks:
```css
code[class*="language-"], pre[class*="language-"] {
```
See: https://github.com/xiaozi/solarized-prism-theme/blob/c6d880bfeae792c1582c3f07071407ac9d17794c/solarized-dark.css#L6

If you have no language specified (which will always be the case with inline code blocks for example) those theme might not work correctly.